include/wx/generic/gridsel.h

```diff
@@ -18,6 +18,9 @@
 
 #include "wx/vector.h"
 
+// Forward declaration
+namespace wxGridPrivate { class PolyPolygon; }
```

Comment:
I wonder if we should call this class `SelectionShape` instead? This name seems to carry more meaning than the current one (especially because it doesn't even always consists of multiple polygons).

Reply:
Done